src/Sarif.Driver.UnitTests/AnalyzeCommandTests.cs

```diff
@@ -209,6 +209,25 @@ public void LoadPdbException()
             );
         }
 
+
+        [Fact]
+        public void FileUri()
+        {
+            Uri uri = new Uri(this.GetType().Assembly.Location);
+
+            var options = new TestAnalyzeOptions()
+            {
+                TargetFileSpecifiers = new string[] { uri.ToString() },
+            };
+
+            ExceptionTestHelper(
+                ExceptionCondition.None,
+                RuntimeConditions.NoErrors,
+                analyzeOptions: options
```

Comment:
Do you want to test that the file:// URI was actually accepted into the list of analysis targets? As written, it looks like the test just verifies that specifying a file:// URI didn't cause an error.